### ISSUE TITLE
How to disable `.cjs` file generation

### ISSUE THREAD

**Issue by ivodolenc** [nuxt/module-builder#91]
Assuming we're only targeting `type: module` packages, how to disable `module.cjs` file generation in `/dist` directory?

I tried this but the file is still generated:

```typescript
// build.config.ts

import { defineBuildConfig } from 'unbuild'

export default defineBuildConfig({
  rollup: {
    emitCJS: false,
    commonjs: false
  }
})
```

### Quick alternatives

Removes it after build via package `script`:

```json
{
  "scripts": {
    "build:module": "nuxt-module-build && rm dist/module.cjs"
  }
}
```

Excludes it via package `files` option:

```json
{
  "files": [
    "dist",
    "!dist/module.cjs"
  ],
}
```

**Comment by pi0** [comment]
Hi. Generated cjs is just a wrapper rather than actual build. Why would you want to remove that?

**Comment by ivodolenc** [comment]
Hi, thanks for fast reply. 

You're right, my version is only ~230 bytes, but if it's not needed, maybe it would be good if there was an option to disable its generation?

If it's not easy to implement, I can use the mentioned alternative.

This is more of a question or asking for advice, I would make a discussion, but I see that the `Discussion` page is not activated on the repo.

**Comment by pi0** [comment]
We can add an option to disable. For the context, this stub makes modules compatible with Nuxt 2. And really harmless as in Nuxt 3, we natively support and pick the mjs format.

**Comment by ivodolenc** [comment]
Yep, I assumed it was for previous versions. Ok, it's not an issue or urgent, but thanks for the info.

**Comment by ivodolenc** [comment]
Hi @pi0, sorry for tagging you here, but I just updated `@nuxt/module-builder` to the latest (0.5.1) version and noticed more extra files when exporting for production.

- Module size before: unpacked size `28.6 kB`
- Module size after (with extra files): unpacked size `35.9 kB`

I ran into some possibly related discussions here (problems with type declaration and nodeResolution in TS) so I believe that's the reason for the extra files. These are `.d.ts`, `.d.mts`, `.d.cts` formats.

The main problem is that this can increase the final version of the package a lot.

I plan to switch all my modules to ESM only so that all other formats (like .cjs, .d.cts) will be unnecessary.

It would be good if we could exclude them from the build, for example explicitly define which formats or extensions we want to generate.

Also, why add extra `.d.mts` files when `.d.ts` works perfectly fine for `esm`?
